go/dpagg/sum_test.go

```diff
@@ -903,6 +903,207 @@ func TestThresholdedResultFloat64(t *testing.T) {
 	}
 }
 
+// Tests ComputeConfidenceInterval that If both bounds are non-negative, then trim any negative interval. Similarly, if both
+// bounds are non-positive then trim any positive interval from the confidence interval.
+func TestSumComputeConfidenceIntervalForInt64PostProcessing(t *testing.T) {
+	// noNoise is set to skip initial argument checking.
+	noNoise := noNoise{}
+	for _, tc := range []struct {
+		opt     *BoundedSumInt64Options
+		confInt noise.ConfidenceInterval // Raw confidence interval.
+		want    noise.ConfidenceInterval // Post-processing.
```

Comment:
nit: `ConfidenceInterval after post-processing.`

(Similar to what you did in Count)